STEERING-COMMITTEE.md

```diff
@@ -139,24 +130,41 @@ Quorum is considered reached when at least half of the members are present.
 
 ## Getting in touch
 
-There are two ways to raise issues to the steering committee for decision:
+There are three ways to raise issues to the steering committee for decision:
 
 1. Emailing the steering committee at
    [steering@knative.team](mailto:steering@knative.team).
    This is a private discussion list to which all members of the committee have
    access.
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
+   
+3. Asking a question on the
+   [#knative-toc-sc-questions](https://cloud-native.slack.com/archives/C04LQCW0C03)
+   channel on [CNCF Slack](https://cloud-native.slack.com/).
 
 ## Composition
 
-The Steering Committee has five (5) seats. Of these five seats, four (4) are elected
-Contributing seats, open to any project contributor, and one (1) is an
-[End User Seat](#end-user-seat) appointed by the Steering Committee.
+The Steering Committee has five (7) seats. These 7 seats are composed as follows:
```

Comment:
```suggestion
The Steering Committee has a minimum of 5 (five) and maximum of (7) seats. These 7 seats are composed as follows:
```

Reply:
Since selected seats are optional, I've made a suggestion to update the text here. We also need to note that these are optional.

Maybe me writing `optional` is bad here. We should encourage SC to fill in those seats.

Reply:
I actually think those seats are _optional_.  If we ended up with 5 people who can regularly attend and move things forward, I don't want to make them feel like they need to get 2 more warm bodies to be performing well.